plugins/inputs/jenkins/jenkins.go

```diff
@@ -244,6 +245,17 @@ func (j *Jenkins) gatherNodesData(acc telegraf.Accumulator) {
 		acc.AddError(err)
 		return
 	}
+
+	// get total and busy executors
+	if nodeResp.TotalExecutors != 0 {
```

Comment:
I think we should add this measurement even when total executors is 0.

Reply:
Updated code as per the suggestion